templates/chatbots/single_chatbot_home.html

```diff
@@ -223,7 +223,7 @@ <h3 class="font-bold text-lg">Link with {{ platform.label }}</h3>
           {% include "chatbots/components/session_table_actions.html" %}
           <div class="absolute top-0 right-0 flex items-center gap-2">
             <button class="btn btn-ghost btn-sm"
-                    hx-get="{% url 'chatbots:sessions-list' request.team.slug experiment.id %}"
+                    hx-get="{% url 'chatbots:sessions-list' request.team.slug experiment.id %}{% if request.GET %}?{{ request.GET.urlencode }}{% endif %}"
```

Comment:
I don't think this will work because the filters update dynamically and this URL will only update on the initial page load